build/helm/cloudcore/templates/rbac_cloudcore.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
   {{- end }}
 rules:
 - apiGroups: [""]
-  resources: ["nodes", "nodes/status", "configmaps", "pods", "pods/status", "secrets", "endpoints", "services","persistentvolumes","persistentvolumeclaims"]
+  resources: ["nodes",  "nodes/status", "serviceaccounts/token", "configmaps", "pods", "pods/status", "secrets", "endpoints", "services","persistentvolumes","persistentvolumeclaims"]
   verbs: ["get", "list", "watch", "create", "update"]
```

Comment:
We still need `node` patch verb, can just help add ut here. Add a new resource as in https://github.com/kubeedge/kubeedge/pull/3466 or add verbs in L12 looks all fine, Add a new resource maybe better?

Reply:
> 

OK, I will do the helm CRD patch. The CRDs in build/cloud dir will need @zivAnyvision‘s help